### ISSUE TITLE
[FEATURE] Support application transparency

### ISSUE THREAD

**Issue by xxxserxxx** [spezifisch/stmps#24]
According to tcell documentation, it _should_ be able to create transparent UIs by using `tcell.SetBackgroundColor()`. I've so far been unsuccessful by following the examples, but haven't spent a lot of time on it.

This would be a nice-to-have option for users via a "-transparent" flag.

**Comment by xxxserxxx** [comment]
@spezifisch, can we take this out for milestone 1.0? It looks like there are [unresolved issues in tview](https://github.com/rivo/tview/pull/395) that might make this difficult, and I think it'd be nice to reach a 1.0 release. I know I created the ticket, but I'd like to kick it down the road; I think it'd touch a lot of files, and it might ultimately get blocked by issues in tview.
